.github/workflows/build-ci-buildbox-images.yaml

```diff
@@ -0,0 +1,82 @@
+name: Build CI Buildbox Images
+run-name: Build CI Buildbox Images
+on:
+  push:
+    paths:
+      - build.assets/Dockerfile
+      - build.assets/Dockerfile-centos7
+      - build.assets/Makefile
+      - build.assets/images.mk
+    branches:
+      - master
+  pull_request:
+    paths:
+      - build.assets/Dockerfile
+      - build.assets/Dockerfile-centos7
+      - build.assets/Makefile
+      - build.assets/images.mk
+    branches:
+      - master
+
+env:
+  REGISTRY: ghcr.io
+  BUILDBOX_BASE_NAME: ghcr.io/gravitational/teleport-buildbox
+
+jobs:
+  buildbox:
+    name: Build Ubuntu Buildbox
```

Comment:
Do we need a Ubuntu and a CentOS 7 box? I thought we were building everything in CentOS now?

Reply:
Yes we use both currently. I think CentOS 7 one is only used for OS compatibility check we added recently, Ubuntu one is still used for everything else.